HLTrigger/Configuration/python/customizer_AlpakaPixelTracking_Phase2_141X.py

```diff
@@ -0,0 +1,247 @@
+            alpaka = cms.untracked.PSet(
+                backend = cms.untracked.string('')
+        )
+    )
+    process.hltESPSiPixelGainCalibrationForHLTSoA = cms.ESProducer('SiPixelGainCalibrationForHLTSoAESProducer@alpaka',
+            appendToDataLabel = cms.string(''),
+            alpaka = cms.untracked.PSet(
+                backend = cms.untracked.string('')
+        )
+    )
+    process.hltESPPixelCPEFastParamsPhase2 = cms.ESProducer('PixelCPEFastParamsESProducerAlpakaPhase2@alpaka', 
+            ComponentName = cms.string("PixelCPEFastParamsPhase2"),
+            appendToDataLabel = cms.string(''),
+            alpaka = cms.untracked.PSet(
+                backend = cms.untracked.string('')
+        )
+    )
+    process.hltOnlineBeamSpotDevice = cms.EDProducer('BeamSpotDeviceProducer@alpaka',
+            src = cms.InputTag('hltOnlineBeamSpot'),
+            alpaka = cms.untracked.PSet(
+                backend = cms.untracked.string('')
+        )
+    )
+    process.siPixelClustersSoA = cms.EDProducer('SiPixelPhase2DigiToCluster@alpaka',
+        mightGet = cms.optional.untracked.vstring,
+        alpaka = cms.untracked.PSet(
+            backend = cms.untracked.string('')
+        )
+    )
+    process.siPixelClusters = cms.EDProducer('SiPixelDigisClustersFromSoAAlpakaPhase2',
+        src = cms.InputTag('siPixelClustersSoA'),
+        clusterThreshold_layer1 = cms.int32(4000),
+        clusterThreshold_otherLayers = cms.int32(4000),
+        produceDigis = cms.bool(False),
+        storeDigis = cms.bool(False)
+    )
+    process.siPixelClusterShapeCache = cms.EDProducer('SiPixelClusterShapeCacheProducer',
+        src = cms.InputTag('siPixelClusters' ),
+        onDemand = cms.bool( False )
+    )
+    process.siPixelRecHitsSoA = cms.EDProducer('SiPixelRecHitAlpakaPhase2@alpaka',
+        beamSpot = cms.InputTag('hltOnlineBeamSpotDevice'),
+        src = cms.InputTag('siPixelClustersSoA'),
+        CPE = cms.string('PixelCPEFastParamsPhase2'),
+        mightGet = cms.optional.untracked.vstring,
+        # autoselect the alpaka backend
+        alpaka = cms.untracked.PSet(
+            backend = cms.untracked.string('')
+        )
+    )
+    process.siPixelRecHits = cms.EDProducer('SiPixelRecHitFromSoAAlpakaPhase2',
+        pixelRecHitSrc = cms.InputTag('siPixelRecHitsSoA'),
+        src = cms.InputTag('siPixelClusters'),
+    )
+
+    process.HLTDoLocalPixelSequence = cms.Sequence(
+         process.siPixelClustersSoA
+        +process.siPixelClusters
+        +process.siPixelClusterShapeCache  # should we disable this? Still needed by tracker muons
+        #+process.siPixelDigis             # not needed when copying digis from sim
+        +process.siPixelRecHitsSoA
+        +process.siPixelRecHits
+    )
+    process.HLTDoLocalStripSequence = cms.Sequence(
+        process.siPhase2Clusters
+    )
+    process.itLocalRecoSequence = cms.Sequence(
+         process.HLTDoLocalStripSequence
```

Comment:
Why do you call this Strips for phase2? Something containing OT would be more appropriate

Reply:
Also I find it confusing you call an OT local reco sequence within `itLocalRecoSequence` (again the naming convention is not respected).

Reply:
this sequence is already defined in https://github.com/cms-sw/cmssw/blob/0cbc4b00b1664496105480e9f561a29ba7c9a68b/HLTrigger/Configuration/python/HLT_75e33/sequences/HLTDoLocalPixelSequence_cfi.py . Its redefinition here is actually not needed, so I removed it from the customiser in 208d97d6f5c4dc677b0ce3d11e5d5db357e97785. The problem of phase2 naming convention will be addressed in a future PR

Reply:
the same is true for https://github.com/cms-sw/cmssw/blob/0cbc4b00b1664496105480e9f561a29ba7c9a68b/HLTrigger/Configuration/python/HLT_75e33/sequences/itLocalRecoSequence_cfi.py, which calls siPhase2Clusters as well. I did not change the structure of already existing sequences because they are called by other paths.

Reply:
> The problem of phase2 naming convention will be addressed in a future PR

@lguzzi please take note of https://github.com/cms-sw/cmssw/pull/45500#issuecomment-2242744894 (the renaming was kindly taken care of by  @AuroraPerego). Please cross-check if you have comments.